### DISCUSSION TITLE
Send batch of images

### DISCUSSION THREAD

**Discussion by Dolise** [devlikeapro/waha#634] [Q&A]
Is there any way to send images in one batch + caption? As i can see API provides to send only 1 image

**Comment by devlikepro** [comment]
You can send multiple image using `POST /api/sendImage`
https://waha.devlike.pro/docs/how-to/send-messages/#send-image-imagesversionspluspng
```json
{
  "chatId": "123@c.us",
  "file": {
    "mimetype": "image/jpeg",
    "filename": "filename.jpg",
    "url": "https://github.com/devlikeapro/waha/raw/core/examples/waha.jpg"
  },
  "session": "default"
}
```

when clients (the app on phone) receive >4 messages with no caption - it'll combine it on UI side

[![patron:PRO](https://img.shields.io/badge/patron-PRO-188a42)](https://waha.devlike.pro/docs/how-to/plus-version/#tiers)
